### ISSUE TITLE
Rebranding

### ISSUE THREAD

**Issue by Oddface** [John-Smith-Modded/JSTR-Modded-1.7.x#155]
We discussed this in https://github.com/John-Smith-Modded/JSTR-1.7.x/issues/129 a little while back and I thought I'd make a separate issue for it. To sum up, we wanted to change the name and branding of the pack to better represent what modded Minecraft is today. We settled on "Compendium: John Smith for Modded Minecraft" as the name. This is the latest iteration of the logo that @Glowstrontium made:
![image](https://cloud.githubusercontent.com/assets/7591428/13665061/d91f0dae-e6aa-11e5-965f-8f1df1549bd7.png)
A reason to make the change now is that we're making some substantial changes to the 1.8 version of the pack, including big overhauls of cobblestone and stone bricks. The plan is to backport these changes to 1.7 with the rebranding. I also think it would be for the best if we got the rebranding done before implementing the new update system (https://github.com/John-Smith-Modded/JSTR-Modded-1.7.x/issues/149)
Thoughts? Potential changes? Practicalities?

**Comment by ArfourFoulkesayke** [comment]
I do like that very much. :+1:

**Comment by Hosed** [comment]
I think I would have just gone with John Smith Modded for the name and left it at that, but this works.

As for the logo, I've never liked the back-drop accents of this one (too much sand stone).  When I think of JS vs JSTR, I think of the stone and brick blocks.  These were the two blocks that led me to remixing JSTR back to legacy until the overwrites were included in the repo.  I would go so far as making my own versions of all the ores before someone else took up the mantle.

Anyways, the rebranding, if I recall, was all about bringing back more JS feel to modded so I would feature those two block types in the back drop over Sandstone Brick.  The five textures that define JS are the Oak Logs, Oak Planks, Stone, Cobble/Stone Brick (toss-up), and White/Gray wool.

**Comment by lesdmark** [comment]
I'm ok with that logo as it is the one we had pretty much settled on ages ago.

**Comment by Oddface** [comment]
We'll also need a new icon for the resource pack. Some [assets](https://github.com/John-Smith-Modded/john-smith-modded.github.io/tree/master/assets/img) for the website will need to be redone as well.

**Comment by Oddface** [comment]
I don't know, I'm kinda changing my mind on the title. Maybe we should just go with "John Smith Modded" like @Hosed said. It just gets the point across more effectively and I guess I don't really see a good reason to have a very different title for this now that it's actually going to be closer to the regular John Smith pack in a lot of ways.

**Comment by ScottKillen** [comment]
I think there was an idea that we needed to separate from the original so we could incorporate and assert some control over the base pack.

But it doesn't really matter to me.

**Comment by Oddface** [comment]
True. That was the idea, and we're still doing that with the new vanilla repos, but the changes are still minor, sticking pretty closely to the standard John Smith look, with a few changes for consistency and some small improvements. I think operating under the John Smith umbrella as an independent entity would still work out just fine.

**Comment by Oddface** [comment]
The big changes are out of the way and the new update system is ready to go, so we only need a new name to be able to release this thing.

**Comment by ScottKillen** [comment]
new name: Compendium? :grinning:

**Comment by Oddface** [comment]
Sure 😁

**Comment by ScottKillen** [comment]
Is this still a thing?

**Comment by lesdmark** [comment]
Is what still a thing?

**Comment by ScottKillen** [comment]
I asked you first! 😆

**Comment by lesdmark** [comment]
I guess not. I only started working on anything again sometime around the end of the year. Any changes from then until now I have either personally done, or approved people's work that was submitted. I also got a 1.12 version released on Curse and am trying to keep it somewhat up to date by getting the person who can do it to push new releases.

**Comment by lesdmark** [comment]
I have only been working on the 1.12 version and am super thrilled that a newer version of modded is on the horizon. It's like painting the Golden Gate Bridge.

**Comment by artdude543** [comment]
Yeah my apologies on the slow pushing and vanishing. Heath issues with myself and parent's hinders that. Then to play catch up with other commitments. 

But the new script is nearly done. Just need to fix some custom code for this pack. Then it'll be shoved on a server with Cron and forgotten about xD 

But again really sorry for being really bad at my part on this.

**Comment by ScottKillen** [comment]
I'm grateful for what you both do.

**Comment by lesdmark** [comment]
> Yeah my apologies on the slow pushing and vanishing. Heath issues with myself and parent's hinders that. Then to play catch up with other commitments.
> But the new script is nearly done. Just need to fix some custom code for this pack. Then it'll be shoved on a server with Cron and forgotten about xD
> But again really sorry for being really bad at my part on this.

It's ok, I just keep patching more mistakes I find all the time, So the later the release the more fixes it contains. :)
